src/platform/datapath_xplat.c

```diff
@@ -49,6 +49,7 @@ CxPlatDataPathInitialize(
     char* EnableXdp = getenv("MSQUIC_ENABLE_XDP");
```

Comment:
Please remove env variable usage, and make sure Windows and Linux both use the flag below.